CMakeLists.txt

```diff
@@ -310,6 +311,29 @@ endif ()
 set(MLPACK_INCLUDE_DIRS ${MLPACK_INCLUDE_DIRS} ${ARMADILLO_INCLUDE_DIRS})
 set(MLPACK_LIBRARIES ${MLPACK_LIBRARIES} ${ARMADILLO_LIBRARIES})
 
+# Download stb_image for image loading.
+if (DOWNLOAD_STB_IMAGE)
+  set(STB_DIR "stb")
+  install(DIRECTORY DESTINATION ${CMAKE_BINARY_DIR}/deps/${STB_DIR})
+  file(DOWNLOAD https://raw.githubusercontent.com/nothings/stb/master/stb_image.h
```

Comment:
I think instead of using always the latest version, we should pick a stable and host the file on mlpack, that way we can control which version is going to be included. @MuLx10 @rcurtin Let me know what you think.

Reply:
@zoq Yes using a stable version would be good. But how do I host file on mlpack?

Reply:
We can upload the file.

Reply:
Where should I upload it?

Reply:
You can just add it to the mlpack.org repo under files/stb/ or something like this.  Once it's merged there the website will automatically rebuild it and it will be available.